ruff/__main__.py

```diff
@@ -0,0 +1,7 @@
+import os
+import sys
+import sysconfig
+
+if __name__ == "__main__":
+    ruff = os.path.join(sysconfig.get_path("scripts"), "ruff")
```

Comment:
Do we need this logic from [maturin](https://github.com/PyO3/maturin/blob/main/maturin/__main__.py#L8), or can it intentionally be omitted?

Reply:
I looked at the maturin logic, but it seems unnecessarily complicated. It finds all possible `scripts` path via all possible schemes (not just the current one), recurses through all files in all subfolders of these `scripts` paths, and strips off all file extensions to check the basename. It originated from a Windows user at PyO3/maturin#1038. But I’ve tested this simple version to work on Windows, so I don’t see why any of that complexity should be needed. Perhaps that user had a weirdly misconfigured Python installation.

Reply:
Sounds good :)